cmd/print.go

```diff
@@ -65,9 +65,12 @@ type PrintRunner struct {
 // converted Gateways and HTTP Routes. The steps include reading from the source,
 // construct ingresses, convert them, then print them out.
 func (pr *PrintRunner) PrintGatewaysAndHTTPRoutes(cmd *cobra.Command, _ []string) error {
+	if pr.providers == nil || len(pr.providers) == 0 {
+		return fmt.Errorf("you must specify at least one provider with --providers")
+	}
```

Comment:
consider using [MarkFlagRequired](https://pkg.go.dev/github.com/spf13/cobra#MarkFlagRequired) instead

Reply:
That's a good idea, thanks!